mirrord/cli/src/verify_config.rs

```diff
@@ -157,7 +165,7 @@ impl TargetType {
             Self::Pod => !(config.copy_target.enabled && config.copy_target.scale_down),
             Self::Job | Self::CronJob => config.copy_target.enabled,
             Self::Service => !config.copy_target.enabled,
-            Self::Deployment | Self::StatefulSet | Self::ReplicaSet => true,
+            Self::Deployment | Self::StatefulSet | Self::ReplicaSet | Self::Workflow => true,
```

Comment:
How come workflow targets don't require a copy_target? Since they actually target a specific node of the workflow graph that ends, like a job?

Reply:
I'm a bit hesitant in requiring it being copy-target just yet because of the amount of customization included in workflows, Right now we can add support for "catching" a running workflow